docs/components/Content/Web.stories.tsx

```diff
@@ -21,7 +21,7 @@ export default {
 const BasicTemplate: ComponentStory<typeof Content> = args => (
   <Content {...args}>
     <Card title="About me">
-      <Content {...args}>
+      <Content type="section" {...args}>
```

Comment:
I'd drop that type down to line 47 so the storybook controls shows that value as well

Reply:
Ahh thanks. Haven't done much with Storybook yet.

Reply:
Done.